src/core.js

```diff
@@ -1076,16 +1076,14 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
 
   /**
    * @description
-   * Set new value to a cell. To change many cells at once, pass an array of `changes` in format `[[row, col, value], ...]` as
-   * the only parameter. `col` is the index of a __visible__ column (note that if columns were reordered,
-   * the current visible order will be used). `source` is a flag for before/afterChange events. If you pass only array of
-   * changes then `source` could be set as second parameter.
+   * Set new value to a cell. To change many cells at once (recommended way), pass an array of `changes` in format
+   * `[[row, col, value],...]` as the first argument.
    *
    * @memberof Core#
    * @function setDataAtCell
-   * @param {Number|Array} row Visual row index or array of changes in format `[[row, col, value], ...]`.
-   * @param {Number} col Visual column index.
-   * @param {String} value New value.
+   * @param {Number|Array} row Visual row index or array of changes in format `[[row, col, value],...]`.
+   * @param {Number} [column] Visual column index.
+   * @param {String} [value] New value.
    * @param {String} [source] String that identifies how this change will be described in the changes array (useful in onAfterChange or onBeforeChange callback).
    */
   this.setDataAtCell = function(row, col, value, source) {
```

Comment:
You've changed jsdoc from `col` to `column` but function parameters and code is not updated.

Reply:
Fixed.